src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'book-store/src/redux/categories/CHECK_STATUS';
+
+const init = [];
+
+export default function categoriesReducer(state = init, action) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return [...state, action.payload];
```

Comment:
- [ ] We need to return the 'Under construction' string here not an array. Kindly fix that.